athlinks-schema.ttl

```diff
@@ -123,12 +139,12 @@ athlinks:ranking a owl:ObjectProperty;
 athlinks:defaultPace
   a owl:ObjectProperty ;
   rdfs:domain athlinks:TimingInterval ;
-  rdfs:range schema:QuantitativeValue .
+  rdfs:range schema:Duration .
```

Comment:
I didn't add changes for that fields
I created fork 
https://github.com/athlinks/rdf-ontology/pull/5/commits/cdcbae5a715ce84deff65688519c7cc5e6ac27ef
you will see my changes